catering/data/knowledge_article.xml

```diff
@@ -0,0 +1,408 @@
+        <div class="o_editor_banner user-select-none o-contenteditable-false lh-1 d-flex align-items-center alert alert-info pb-0 pt-3" data-oe-role="status">
+            <i class="o_editor_banner_icon mb-3 fst-normal" data-oe-aria-label="Banner Info">💡</i>
+            <div class="o_editor_banner_content o-contenteditable-true w-100 px-3">
+                <p><b>In the Planning App</b><br/></p>
+                <ul>
+                    <li>
+                        <p>Input all <b>employees</b> into the configuration. (Note: These employees are for planning purposes, are <b>not Odoo users</b>, and <b>do not generate extra Odoo costs</b>.)</p>
+                        <ul>
+                            <li>
+                                <p>Set an <b>hourly cost</b> for each employee if you plan to track labor costs.</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <p>Create <b>specific roles</b> for your project (e.g., Waiter, Maître d'hôtel, Kitchen Porter).</p>
+                    </li>
+                    <li><p>Assign employees to one or more of these specific roles.</p></li>
+                    <li>
+                        <p>Create <b>"Shift Templates"</b> to easily build the planning schedule for your project. (This streamlines the creation of recurring or common shift patterns.)</p>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <p style="text-align: justify;">
+            You can access the <b>Planning tool</b> via a button in the <b>top bar</b> of the Project app. This tool allows you to <b>efficiently prepare and schedule</b> the necessary workers for your event. Simply
+            <b>select the desired shift</b> on the corresponding dates to create it.
+        </p>
+        <div data-oe-role="status" class="o_editor_banner user-select-none o-contenteditable-false lh-1 d-flex align-items-center alert alert-warning pb-0 pt-3">
+            <i data-oe-aria-label="Banner Warning" class="o_editor_banner_icon mb-3 fst-normal">⚠️</i>
+            <div class="o_editor_banner_content o-contenteditable-true w-100 px-3">
+                <p>To see the pre-existing plan and the shifts that still require scheduling, please select an additional 40 days, now that the demo data is uploaded.</p>
+            </div>
+        </div>
+        <div class="o_editor_banner user-select-none o-contenteditable-false lh-1 d-flex align-items-center alert alert-success pb-0 pt-3" data-oe-role="status">
+            <i class="o_editor_banner_icon mb-3 fst-normal" data-oe-aria-label="Banner Success">✅</i>
+            <div class="o_editor_banner_content o-contenteditable-true w-100 px-3">
+                <p>You need to duplicate several time the same shift? press Ctrl and move the shift, it will duplicate it automatically.</p>
```

Comment:
```suggestion
                <p>You need to duplicate several times the same shift? Press Ctrl and move the shift, it will duplicate it automatically.</p>
```